src/lib389/lib389/cli_conf/plugins/memberof.py

```diff
@@ -139,7 +165,29 @@ def _add_parser_args(parser):
     parser.add_argument('--launchfixup', choices=['on', 'off'], type=str.lower,
                         help='Specify that if the server disorderly shutdown (crash, kill,..) then '
                              'at restart the memberof fixup task is launched automatically')
+    parser.add_argument('--specific-group-oc', nargs='+',
+                        help='Set objectclasses for the specific groups to include/exclude. '
+                              'Otherwise all other groups will be excluded. Note, this replaces '
+                              'all existing values. To add a new value, use the "add-attr" command. '
+                              '(memberOfSpecificGroupOC)')
+    parser.add_argument('--specific-group-filter', nargs='+',
+                        help='Specifies a filter for groups to include. This adds to the existing values. Otherwise all other groups will be excluded (memberOfSpecificGroup)')
+    parser.add_argument('--exclude-specific-group-filter', nargs='+',
+                        help='Specifies a filter for groups to exclude. Otherwise all other groups will be included. '
+                             'Note, this replaces all existing values. To add a new value, use the "add-attr" command. '
+                             '(memberOfExcludeSpecificGroup)')
```

Comment:
```suggestion
                             '(memberOfExcludeSpecificGroupFilter)')
```

Couple of old typos :)